gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -354,6 +355,10 @@ message TripUpdate {
     // populated and will be ignored by consumers.
     // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
     optional string start_time = 3;
+    // Specifies the shape of the vehicle travel path of the trip. See definition of trips.shape_id in (CSV) GTFS.
+    // Refers to a shape defined in the (CSV) GTFS or in the (protobuf) real-time feed. 
```

Comment:
Should this include the note that "the order of stops (stop_sequences) in the TripUpdate must remain the same as (CSV) GTFS" (from the "Information" box on this field in the google doc)?

Reply:
I think what your referring to is
> You can also define new shapes for existing (CSV) GTFS trips by populating this shape_id field with a shape ID pointing to a new Shape entity using the GTFS-NewShapes extension described below. However, the order of stops (stop_sequences) in the TripUpdate must remain the same as (CSV) GTFS. If you want to represent a trip that differs in stop_sequences from (CSV) GTFS (e.g., adding new stops), you must instead create a new Trip entity using the GTFS-NewTrips extension described below.

This is interesting - just confirming our plan if we have a new shape and stops are skipped? Are we planning to use the existing trip id and marking skipped stops (as we are now, I think this is the plan) or are we planning to use `GTFS-NewTrips` to indicate what stops will actually be visited?

Reply:
It should be okay to stick with our plan because we are only skipping stops. Once we do full detours, maybe we will use Trip even in the case where no stops are added.